src/fragments/lib-v1/storage/js/copy.mdx

```diff
@@ -7,19 +7,16 @@ success.
 
 </Callout>
 
-To copy a file, you need to provide a source key in `source` and a destination key in `destination`. `accessLevel` can be 'guest' | 'private' | 'protected' but defaults to 'guest'.
+To copy a file, you need to provide a source key in `source` and a destination key in `destination`. `accessLevel` can be 'guest', 'private', or 'protected', defaulting to 'guest'.
 
 ```javascript
 import { copy } from 'aws-amplify/storage';
 
 const copyFile = async () => {
 	try {
 		const response = copy({
-			source: {
-				key: 'srcKey',
-				accessLevel: srcLevel
-			},
-			destination: { key: 'destKey, accessLevel: destLevel },
+			source: { key: 'srcKey' },
```

Comment:
I think we can provide a minimalistic example first

Reply:
I think you can keep the existing example but make `accessLevel` to be optional.